engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -95,6 +97,19 @@ public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
 
+    @Override
+    public void setUri(String nodeUri) {
```

Comment:
This sounds.. way too hacky. Waaaaaay too hacky.

Counter proposal : How about making the URI final, and setting it in the constructor?